lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -96,8 +100,19 @@ def safe_method(method_name, node)
           end
         end
 
+        def check_localtime(node)
+          p = node
```

Comment:
`p` means `parent` and we use such notation in `extract_method_chain` (in both TimeZone and Date cops). Do we need to rename them all?

And we cannot omit it at all (and use just `node`) because we need initial `node` to register offense if any.